resources/authgear/templates/en/web/authflowv2/__select_input.html

```diff
@@ -2,7 +2,7 @@
 
 {{ $options := prepend $.Options (dict
   "triggerLabel" (include "v2.component.select-input.default.not-provided-label" nil)
-  "label" (include "v2.component.select-input.default.unset-label" nil)
```

Comment:
Please also help remove `v2.component.select-input.default.unset-label` from the translation files

Reply:
Ok, completed 👌